better_bookmarks/js/content_script.js

```diff
@@ -0,0 +1,28 @@
+var port = chrome.runtime.connect();
+
+window.addEventListener("message", (event) => {
+    if (event.source != window)
+        return;
+
+    if (event.data.type && (event.data.type == "FROM_PAGE")) {
+        console.log("Content script received: " + event.data.text);
+        port.postMessage(event.data.text);
+    }
+}, false);
+
+const title = document.getElementsByTagName("title")[0];
+const metas = document.getElementsByTagName("meta");
+const url = document.URL;
+for (let i = 0; i < metas.length; i++) {
+    const meta = metas[i];
+    if (meta.name.toLowerCase() == "description"){
```

Comment:
I found the following quote regarding attribute casing, *"HTML5 attributes are case insensitive and may be written in all uppercase or mixed case, although the most common convention is to stick with lowercase."*

I think the ```toLowerCase()``` method call should be removed since it isn't really needed in my opinion.

Reply:
Just as an example but w3schools.com uses the tag name Description. If I don't use `toLowerCase()` then it won't get the description tag for w3schools.com

Reply:
Okay, fair enough.  I didn't think anyone strayed away from the standard of "description" in all lowercase, but good job finding this out.  Add a comment above though, mentioning that some sites don't adhere to the standard such as W3Schools.